### ISSUE TITLE
Simplify the API response

### ISSUE THREAD

**Issue by sjparkinson** [Financial-Times/tako#93]
We currently return...

```json
{
  "full_name": "Financial-Times/custard",
  "name": "custard",
  "url": "https://github.com/Financial-Times/custard",
  "owner": "Financial-Times",
  "topics": [
    "customer-products",
    "serverless"
  ]
}
```

Could we instead return...

```json
{
  "repository": "Financial-Times/custard",
  "topics": [
    "customer-products",
    "serverless"
  ]
}
```

**Comment by adambraimbridge** [comment]
After IRL discussion we landed on this: 
```{
  "name": "custard",
  "owner": "Financial-Times",
  "topics": [
    "customer-products",
    "serverless"
  ]
}```

**Comment by ker-an** [comment]
Issue resolved via PR #102
